packages/editor/src/store/selectors.js

```diff
@@ -51,44 +38,8 @@ export const INSERTER_UTILITY_HIGH = 3;
 export const INSERTER_UTILITY_MEDIUM = 2;
 export const INSERTER_UTILITY_LOW = 1;
 export const INSERTER_UTILITY_NONE = 0;
-const MILLISECONDS_PER_HOUR = 3600 * 1000;
-const MILLISECONDS_PER_DAY = 24 * 3600 * 1000;
-const MILLISECONDS_PER_WEEK = 7 * 24 * 3600 * 1000;
 const ONE_MINUTE_IN_MS = 60 * 1000;
-
-/**
- * Shared reference to an empty array for cases where it is important to avoid
```

Comment:
This was a pretty nice comment which still applies to the new `EMPTY_OBJECT`. I think it should have been kept.